src/runtime_src/core/edge/user/shim.h

```diff
@@ -232,8 +234,102 @@ class shim {
     {
       m_shim->xclExecBuf(cmd->get_xcl_handle());
     }
+    std::shared_ptr<xrt_core::graph_handle>
+    open_graph_handle(const char* name, xrt::graph::access_mode am) override
+    {
+      return std::make_shared<graph_object>(m_shim, m_uuid, name, am);
```

Comment:
Is the graph_object / handle shared, or is it owned by graph_impl?  If latter it should be a std::unique_ptr created and returned here.

Reply:
changed to unique_ptr